src/main/drivers/serial_uart_impl.h

```diff
@@ -47,7 +47,7 @@
 #define UART_RX_BUFFER_SIZE     128
 #endif
 #ifndef UART_TX_BUFFER_SIZE
-#define UART_TX_BUFFER_SIZE     256
+#define UART_TX_BUFFER_SIZE     1024
```

Comment:
Maybe better to only increase this if `USE_MSP_DISPLAYPORT` - the current default seems to be fine for all other implementations.
Also, have you tested with other values (e.g. 512, 768) to see if they are sufficient?

Reply:
What you said makes a lot of sense. My FC has a little failure and I am repurchasing. I will complete the test as soon as possible in the next few days: use a large number of osd elements and a small number of MSP commands at the same time to perform "overflow" tests on different FIFO SIZEs.

Reply:
I also tested ‘’‘’UART_TX_BUFFER_SIZE=256/512/768/1024/1280/2048‘’‘’.
Add almost full-screen osd elements. From power-on, dump the msp_displayport datas of different sizes, and then compare them with Winmerge software to determine whether there is any missing data.
Test Results:
         256/512/768/1024: All data will be lost when starting the interface. After 1024 size, osd elements are not lost, but osd elements under other sizes will be lost.
         1280: No data is lost.
         2048: The same result as 1280.